src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig

```diff
@@ -122,6 +122,7 @@
 {%- block radio_widget -%}
   <input
     type="radio" {{ block('widget_attributes') }}{% if value is defined %} value="{{ value }}"{% endif %}{% if checked %} checked="checked"{% endif %} />
+  <i class="form-check-round"></i>
```

Comment:
Weird, it will break part of code :thinking:  it should be surrounded by a div?

Reply:
Why would it break something ?

Tests didn't find anything, also it's aiming every basic forms, so they all get this modification, I couldn't find anything broken

It can't be surrounded by a div and it's not necessary to as it would be the same behavior with a div or without, anyway the input is hidden as it's a custom radio component

Reply:
I'm not used to it,  I'm always afraid to have a field related to another one without being surrounded :sweat_smile:

Reply:
Well, in this case, the icon is related to the input just next, and the field is surrounded way upper in the block including this input + the icon